R/xcmsSet.R

```diff
@@ -253,6 +253,18 @@ split.xcmsSet <- function(x, f, drop = TRUE, ...) {
     lcsets <- vector("list", length(levels(f)))
     names(lcsets) <- levels(f)
 
+    ## get the phenoData and all other parameters we want to pass
+    ## down to the splitted objects
+    pd <- phenoData(x)
+    dataCor <- x@dataCorrection
+    suppressWarnings(
```

Comment:
Which warnings are expected here ? Please add to the comment, same three lines below.